README.md

```diff
@@ -768,6 +768,40 @@ your **OpenWISP WiFi Login Pages** service.
 With the right configuration, the error messages coming from freeradius or
 the captive portal will be visible to users on **OpenWISP WiFi Login Pages**.
 
+### Allowing users to manage account from the Internet
+
+With following configuration **OpenWISP WiFi Login Pages** can allow users to
+login and manage there account even when they are not connected to the
+WiFi service.
```

Comment:
I would change this text to make it clear that the problem is that the captive portal will most probably not be reachable from the public internet, so login and logout requests will hang.

Is not that the web app cannot work from the internet, it works but those requests hang making it look like it doesn't work (but if you wait long enough the request will time out and flow will continue).